src/app/components/mno-locale-config/mno-locale-config.svc.coffee

```diff
@@ -31,15 +36,15 @@ angular.module 'mnoEnterpriseAngular'
       found = path.match(re)
 
       # Ex found: ["/en/dashboard/", "en", index: 0, input: "/en/dashboard/"]
-      locale = found[1] if found?
+      locale = filterAvailableLocales(found[1]) if found?
 
       $q.resolve(locale)
```

Comment:
What happens here if locale has not been found?

Reply:
It returns `undefined` so it will fallback to another locale.

See `@configure`:
```coffee
       $q.all(url: localeFromUrl(), user: localeFromUser()).then(
         (response) ->
           if l = response.url || response.user
             setLocale(l)
           else
             setFallbackStack($translate.use())
       )
```